gtk4/src/auto/header_bar.rs

```diff
@@ -36,7 +36,7 @@ impl HeaderBar {
         unsafe { Widget::from_glib_none(ffi::gtk_header_bar_new()).unsafe_cast() }
     }
 
-    pub fn get_decoration_layout(&self) -> Option<glib::GString> {
+    pub fn get_decoration_layout(&self) -> glib::GString {
```

Comment:
This should be an option per your upstream patch

Reply:
True, upstream is correct. Thanks!